app/helpers/users_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module UsersHelper
+  def gravatar_for user, size: Settings.gravatar_size
```

Comment:
what is the difference between def gravatar_for user, size: Settings.gravatar_size and def gravatar_for user, size ???

Reply:
sử dụng Settings.gravatar_size dễ cho việc quản lí sau này như update, ...